airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -92,16 +92,22 @@ public class ConfigRepository {
   private final ConfigPersistence persistence;
   private final ExceptionWrappingDatabase database;
   private final ActorDefinitionMigrator actorDefinitionMigrator;
+  private final StandardSyncPersistence standardSyncPersistence;
 
   public ConfigRepository(final Database database) {
-    this(DatabaseConfigPersistence.createWithValidation(database), database, new ActorDefinitionMigrator(new ExceptionWrappingDatabase(database)));
+    this(DatabaseConfigPersistence.createWithValidation(database), database, new ActorDefinitionMigrator(new ExceptionWrappingDatabase(database)),
+        new StandardSyncPersistence(database));
   }
 
   @VisibleForTesting
-  ConfigRepository(final ConfigPersistence persistence, final Database database, final ActorDefinitionMigrator actorDefinitionMigrator) {
+  ConfigRepository(final ConfigPersistence persistence,
```

Comment:
It may make sense to just make this constructor the one, public constructor in order to better support dependency injection.

Reply:
I agree that's how we should be doing it eventually, for now, I'd rather not to avoid extra breaking changes with cloud.